packages/projection-typeorm/src/types.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
there doesn't seem to be any `any` in the code 
```diff
- /* eslint-disable @typescript-eslint/no-explicit-any */

Reply:
96ec056b6ae41868b0729e4e1b319a45c4f9f028